transcom/mymove
# INT- B-20288: Estimated/Actual Departure date labels for Office PPMShipmentInfo/SSW

## [B-20288](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A972139)

## Summary

**INTEGRATION**

This PR updates the appropriate display of PPM move date based on if ACTUAL move date is available for Office PPM details view and PDF SSW. Currently, it displays the estimated move date regardless if actual move date is available. This PR will make the display more accurate based on current state.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

The logic for this PR is simple. Display "Actual Departure date" when ppm. actualMoveDate is set regardless if ppm.estimatedDepartureDate is set. PPM status was not used. "Estimated Departure date" is display when actualMoveDate is not set.  

1. Login as customer and submit PPM. Make note of the estimated date specified. 
2. Login as Office user and view move details for PPM. It shall display estimated date set in step 1.
![Screenshot 2024-07-22 at 12 04 16 PM](https://github.com/user-attachments/assets/f85b6ded-05ac-422d-abe3-eb320900ce97)
3. Fill in required info and submit move. 
4. Login as customer again and "upload documents" for review. Make note of the actual move date used.
5. Login as Office user and go to move details of PPM. Verify "Estimated Departure date" is now "Actual Departure date" using the value set in step 4.
6. review and approve documents. 
7. Download AOA/Payment packet and verify "Actual delivery date" label is present under Shipments section. It must match what was selected in step 4.
<img width="630" alt="Screenshot 2024-07-22 at 12 42 33 PM" src="https://github.com/user-attachments/assets/bc189365-699c-4935-a15c-8b2ef614dbd3">

### Frontend
